templates/templates/base.template

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I think this should be in `/templates/preview/base.template` instead of `/templates/base.template`

Having it in the root templates directory implies it should be used by everything and could be misleading later.

Reply:
changed